v2/internal/attacktechniques/aws/discovery/ses-enumerate/main.tf

```diff
@@ -0,0 +1,52 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "aws" {
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  default_tags {
+    tags = {
+      StratusRedTeam = true
+    }
+  }
+}
+
+locals {
+  resource_prefix = "stratus-red-team-ses-enumerate"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_iam_role" "ses_enumerate_role" {
```

Comment:
Any specific reason why we need this role?

Reply:
Similarly to the other discovery attack techniques, we just wanted a role to attach the `AmazonSESReadOnlyAccess` policy to

Reply:
Which one(s) specifically? I think some other attacks create a role specifically to get access denies errors